graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -44,6 +51,138 @@ def _are_filter_fields_uniquely_indexed(filter_fields, unique_indexes):
     return False
 
 
+def _try_obtain_parameter_value(argument_name, parameters):
```

Comment:
nit: prefixing the name with `try` is a common existing convention for functions where `None` or `False` is returned upon failure, and I think this would be confusing to the reader. I'd suggest trying to find a different name.

Here is an example function that uses a `try` prefix per the convention I mentioned: https://docs.microsoft.com/en-us/dotnet/api/system.int32.tryparse?view=netframework-4.8

Reply:
I've since deleted this function, since Leon mentioned how we didn't need this.